libs/barista-components/stacked-series-chart/src/stacked-series-chart.ts

```diff
@@ -273,6 +298,11 @@ export class DtStackedSeriesChart implements OnDestroy {
 
     merge(this._shouldUpdateTicks, this._resizer.change())
       .pipe(
+        tap(() => {
+          // Recalculate every time the size changes
+          this._isAnyLabelOverflowing();
+          this._changeDetectorRef.detectChanges();
```

Comment:
This is for the "auto" labelAxisMode which every time there is a resize needs to recalculate if there is any label that is overflowing its assigned space on the css grid, if so we will apply the compact mode(labels rotated 45º to fit more). The function _isAnyLabelOverflowing is used for the getter _labelAxisCompactModeEnabled  and it does not recalculate once a resize happens since it's something external, that's why i'm forcing it here.

Reply:
If we know the current mode of the ticks, which I think we do, in the `_isAnyLabelOverflowing` we could trigger `_shouldUpdateTicks` if it changed and avoid calling de `_changeDetectorRef`

Reply:
I'll test whether we can get rid of this here.

Reply:
@subarroca This is a good idea - @dani7cl could you test this idea please :) ?

Reply:
Hey @ffriedl89 we've tried some pair programming with @subarroca and since it depends on the offsetWidth of the element the detectChanges needs to be applied, any other suggestion?

Reply:
I think, this exception is fine here. I just wanted to make sure. We had similar issues in other charts due to the fact, that in order to do calculations you need a render cycle and therefore need to render twice.